src/phases/tpch/normalized/Q17.yml

```diff
@@ -0,0 +1,36 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 17 (see http://tpc.org/tpc_documents_current_versions/pdf/tpc-h_v3.0.0.pdf) against the normalized schema. Using an 'executionStats' explain causes each command to run its execution plan until no
+  documents remain, which ensures that the query executes in its entirety.
+
+batchSize: &batchSize 101  # The default batch size.
+
+query17Brand: &query17Brand {^Parameter: {Name: "Query17Brand", Default: "Brand#23"}}
+query17Type: &query17Container {^Parameter: {Name: "Query17Container", Default: "MED BOX"}}
```

Comment:
[NIT] Is this `query17Type` a typo or intentional as an alias of `query17Container`? 
Ditto in denormalized/Q17.yml

Reply:
Oops. Thank you for catching this! Fixed it.